test/browser/home-page.test.jsx

```diff
@@ -0,0 +1,37 @@
+var should = require('should');
+var sinon = window.sinon;
+var React = require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var stubContext = require('./stub-context.jsx');
+var HomePage = require('../../pages/home.jsx');
+
+describe("HomePage", function() {
+  var blogSection;
+
+  beforeEach(function() {
+    blogSection = stubContext.render(HomePage.BlogSection);
+  });
+
+  describe("blogSection", function() {
```

Comment:
@toolness I have this `describe` inside of the `describe("HomePage"`, does it help solving [the issue you mentioned](https://github.com/mozilla/teach.webmaker.org/pull/992/files#r31710108)? Or I could remove this inner one and rename the outer `describe` from `HomePage` to `HomePage.BlogSection`